app/assets/stylesheets/_buttons.scss

```diff
@@ -1,265 +1,279 @@
-    -webkit-transform: translate3d(0, $pixels ,0);
-    transform: translate3d(0, $pixels ,0);
+  -webkit-transform: translate3d(0, $pixels ,0);
+  transform: translate3d(0, $pixels ,0);
 }
 
 
 .tabs-style-circle {
-    .tabs-row {
-        padding-top: 50px;
-        padding-bottom: 50px;
-    }
-	overflow: visible;
-    li::before {
-        position: absolute;
-        top: 50%;
-        left: 50%;
-        margin: -60px 0 0 -60px;
-        width: 120px;
-        height: 120px;
-        border: 1px solid $medium-green;
-        border-radius: 50%;
-        content: '';
-        opacity: 0;
-        @include circle-transition();
-    }
-    li {
-        a {
-            color: $charcoal;
-        }
-    }
-    li.tab-current {
-        a {
-            color: $medium-green;
-            span {
-                //@include translate-vertical(4px);
-            }
-        }
-        &::before {
-            opacity: 1;
-            -webkit-transform: scale3d(1,1,1);
-            transform: scale3d(1,1,1);
-        }
-        .icon::before {
-            @include translate-vertical(-4px);
-        }
+  .tabs-row {
+    padding-top: 50px;
+    padding-bottom: 50px;
+  }
+
+  overflow: visible;
+  li::before {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    margin: -60px 0 0 -60px;
+    width: 120px;
+    height: 120px;
+    border: 1px solid $medium-green;
+    border-radius: 50%;
+    content: '';
+    opacity: 0;
+    @include circle-transition();
+  }
+
+  li {
+    a {
+      color: $charcoal;
     }
+  }
+
+  li.tab-current {
     a {
-        overflow: visible;
-        color: $charcoal;
-        font-weight: 400;
-        font-size: 0.9em;
-        line-height: 1.1;
-        @include cubic-bezier-color();
-        span {
-            display: inline-block;
-            @include cubic-bezier-transform();
-        }
+      color: $medium-green;
+      span {
+        //@include translate-vertical(4px);
+      }
+    }
+
+    &::before {
+      opacity: 1;
+      -webkit-transform: scale3d(1,1,1);
+      transform: scale3d(1,1,1);
     }
+
     .icon::before {
-        @include cubic-bezier-transform();
-        display: block;
-        margin: 0;
-        pointer-events: none;
+      @include translate-vertical(-4px);
     }
-    a:hover, a:focus {
-        color: $medium-green;
+  }
+
+  a {
+    overflow: visible;
+    color: $charcoal;
+    font-weight: 400;
+    font-size: 0.9em;
+    line-height: 1.1;
+    @include cubic-bezier-color();
+    span {
+      display: inline-block;
+      @include cubic-bezier-transform();
     }
+  }
+
+  .icon::before {
+    @include cubic-bezier-transform();
+    display: block;
+    margin: 0;
+    pointer-events: none;
+  }
+
+  a:hover, a:focus {
+    color: $medium-green;
+  }
 }
 
 @media #{$small-only}  {
-	.tabs-style-circle li::before {
-		margin: -40px 0 0 -40px;
-		width: 80px;
-		height: 80px;
-	}
+  .tabs-style-circle li::before {
+	  margin: -40px 0 0 -40px;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered height, margin, width